### ISSUE TITLE
Keep dns.txt up to date

### ISSUE THREAD

**Issue by stekaa** [micahsnyder/cvdupdate#2]
Thanks a lot for your script. It works fine.

It would be nice to keep the file **dns.txt** on local mirror up to date. This file contains version numbers of main, daily and bytecode. So after downloading the files successfully your script could store current version information in this file. Or just copy it from clamav.net (if every download worked fine).

Some common check scripts (on clients) use dns.txt to check if clamav is up to date. Especially clients that are only allowed to access local clamav mirror need this information.

Short version: the replaced script did it ;-)

Thank you in advance 
stekaa

**Comment by micahsnyder** [comment]
Hi @stekaa 

I'm not sure if I understand what you're suggesting.  

Are you suggesting a feature so that FreshClam clients which use a Private Mirror wouldn't need DNS and could instead get the versions from like `https://_yourmirror_/dns.txt`?  

If so, a simpler solution may be to disable using DNS in the FreshClam clients by running with `freshclam --no-dns`?  
You could also (and this is a bit of a hack) set `DNSDatabaseInfo no` in the `freshclam.conf` files for the clients.  It's a hack because it doesn't actually disable DNS, but when it tries to query "no" it fails, and falls back to HTTP mode.  
In both cases, it will query the database version by downloading the 512-byte CVD header and inspecting the version there before deciding if it needs to download the whole file to update.

Regards,
Micah

**Comment by stekaa** [comment]
No, that's not what I meant. Sorry for my bad explanation. Mirroring and Freshclam work fine.

I use monitoring checks (like Icinga, Nagios, CheckMK) to check if the version on Client is up to date. These checks loading dns.txt from local mirror and compare version number to current installed daily. 

As a workaround I copy original dns.txt from clamav to my local mirror regularly. But it would by nicer to only copy it if the download of all files to my mirror was successful. Thats why I suggested including it in the script.

**Comment by micahsnyder** [comment]
How's this? https://github.com/micahsnyder/cvdupdate/pull/6

**Comment by stekaa** [comment]
Worked for me. Thanks again.
